spec/features/add_flickr_photo_to_card_spec.rb

```diff
@@ -0,0 +1,126 @@
+require "rails_helper"
+require "support/helpers/login_helper.rb"
+require "support/helpers/wait_for_ajax.rb"
+include LoginHelper
+include WaitForAjax
+
+describe "Add Flickr photo to card" do
+  let!(:user) { create(:user) }
+  let!(:block) { create(:block, user: user) }
+  let(:photo) { page.find("#thumbnail_0") }
+  let(:start_search) do
+    page.find("#flickr_button").click
+    click_button 'GO'
+  end
+
+  before do
+    login_with("test@test.com", "12345", "Войти")
+    visit new_card_path
+  end
+
+  describe 'when user visits new card page', js: true do
+
+    it "shows Search Flickr button" do
+      expect(page).to have_content('Search Flickr')
+    end
+
+    it "remote_image_url textfield is hidden and blank" do
+      expect(page.find(".remote_url").visible?).to eq(false)
+    end
+
+    it "loading indicator is hidden" do
+      expect(page.find(".loader").visible?).to eq(false)
+    end
+
+    it 'search form is hidden' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.